wdl/PETestChromosome.wdl

```diff
@@ -101,36 +101,54 @@ workflow PETestChromosome {
           discfile_idx = discfile_idx,
           include_list = samples,
           prefix = basename(split),
+          ref_dict = ref_dict,
           common_model = false,
-          tabix_retries = tabix_retries,
           sv_pipeline_docker = sv_pipeline_docker,
           runtime_attr_override = runtime_attr_petest
       }
-      call tasks02.SplitCommonVCF as SplitCommonVCF {
-        input:
-          vcf = split,
-          cnv_size_cutoff = common_cnv_size_cutoff,
-          sv_pipeline_docker = sv_pipeline_docker,
-          runtime_attr_override = runtime_attr_split_vcf
-      }
+    }
+  }
+
+  if (!allosome) {
```

Comment:
Some of these changes might conflict with my proposed changes to support empty shards in https://github.com/broadinstitute/gatk-sv/pull/54 -- just an fyi. It'd be nice if we can make sure the resulting code after merges doesn't error on empty shards.

Reply:
Yes I'll do a rebase of this branch after yours is merged and make sure the empty shard check is in place